### PR TITLE
Fixed defender's objective tracking flag carrier ID

### PR DESCRIPTION
**Pull request by alaswell**
Removed DEFENDERS_OBJECTIVE_ONFLAG and DEFENDERS_OBJECTIVE_ONCARRIER and all associated updates/calls from base_id.lua
	- defenders objective should point to cap always to avoid tracing the flag carrier 
fortressforever/fortressforever#58

### CODE REVIEW

**Comment by alaswell** [comment]
This is ready for review...again. 

The base_id file has been cleaned up and all the default ID maps have been updated to include the new system 
including ff_palermo which has been reverted from base_id_new.lua per fortressforever/fortressforever#60

**Comment by alaswell** [comment]
Crap, yeah, nice catch...thought I fixed that but apparently not

File: maps/ff_palermo.lua [hunk 1]
```diff
@@ -1,20 +1,73 @@
...
+				flag:Remove()
+			end
+		end
+	end	 
+	
+	flags_set_team( attackers )
+	
+	ATTACKERS_OBJECTIVE_ENTITY = GetEntityByName( "cp"..phase.."_flag" )
+	UpdateDefendersObjective()
+	UpdateTeamObjectiveIcon( GetTeam(attackers), ATTACKERS_OBJECTIVE_ENTITY )
```
Comment (squeek502) [inline maps/ff_palermo.lua]: All of this copied code from base_id can be avoided by doing

```lua
local startup_base = startup
function startup()
    startup_base()

    -- palermo specific stuff here
end
```

See http://forums.fortress-forever.com/showthread.php?t=23070 for a more detailed explanation.

**Comment by squeek502** [comment]
Thanks for doing all of this by the way. I'll just continue to nitpick until the SVN situation is resolved and then merge it. :smile:

File: maps/ff_palermo.lua (hunk 1 above)
Comment (alaswell) [inline maps/ff_palermo.lua]: that's awesome!!

**Comment by alaswell** [comment]
haha no worries :smile:

**Pull request merged**